src/MultiFlexi/CronScheduler.php

```
@@ -34,46 +34,44 @@ public function scheduleCronJobs(): void
 
-                    $cron = new CronExpression($runtemplateData['cron']);
+                $cron = new CronExpression($runtemplateData['cron']);
 
-                    $startTime = $cron->getNextRunDate(new \DateTime(), 0, true);
+                $startTime = $cron->getNextRunDate(new \DateTime(), 0, true);
 
-                    if (empty($runtemplateData['delay']) === false) {
-                        $startTime->modify('+'.$runtemplateData['delay'].' seconds');
-                        $jobber->addStatusMessage($emoji.' Adding Startup delay  +'.$runtemplateData['delay'].' seconds to '.$startTime->format('Y-m-d H:i:s'), 'debug');
-                    }
+                $scheduleAt = $startTime->format('Y-m-d H:i:s');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$scheduleAt'. (undefined)

(UnusedLocalVariable)